gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.6-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.8-rc-1-all.zip
```

Comment:
4.8-rc-2 is already releases. Not sure if we need the upgrade to a RC version in the first place. It's up to you.

Reply:
Yup, downgraded to 4.7.